globus_automate_client/helpers.py

```diff
@@ -0,0 +1,33 @@
+import typing as t
+
+
+def merge_lists(*args) -> t.Optional[t.List]:
```

Comment:
This func is a little confusing and I think that's because it's being flexible to multiple inputs. What is it being specifically used for? I'm wondering if we can pin down exactly what we need it to handle so that we can make its args + implementation more rigid.

Reply:
I kinda agree this is a bit wonky. I set up the input signature for this for a very particular use case of wanting to merge lists where some were lists as params to a function and some were potentially in kwargs to that function. So, I wanted to just be able to pass the list of values concisely in the caller and have this merge those lists.

This was all driven by trying to use kwargs for backward compatibility where we were changing the name of some parameters.

So, bottom-line, I agree this is on the spectrum of wonky to "very purpose specific". We can further discuss towards something more rigid/better defined if you'd like.